email_alert_service/models/message_processor.rb

```diff
@@ -49,12 +49,12 @@ def trigger_email_alert(document)
     EmailAlert.new(document, @logger).trigger
   end
 
-  def tagged_to_topics?(document)
-    details = document.fetch("details", {})
-    if details.has_key?("tags") && details.fetch("tags").has_key?("topics")
-      details.fetch("tags").fetch("topics").any?
-    else
-      false
+  def email_alerts_supported?(document)
+    document_tags = document.fetch("details", {}).fetch("tags", {})
+    supported_tag_names = ["topics", "policy"]
+
+    supported_tag_names.any? do |tag_name|
+      !(document_tags[tag_name].nil? || document_tags[tag_name].empty?)
```

Comment:
Nvm this comment, since this PR has already been merged. We'll take care of it later.